tests/test_lhe_reader.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Filename: test_lhe_reader.py
+
+__author__ = "Johannes Schumann"
+__copyright__ = "Copyright 2020, Johannes Schumann"
+__credits__ = []
+__license__ = "MIT"
+__maintainer__ = "Johannes Schumann"
+__email__ = "jschumann@km3net.de"
+__status__ = "Development"
+
+import unittest
+import pylhe
+from os.path import dirname, join
+
+TEST_DATA_DIR = join(dirname(__file__), 'test_data')
+
+class TestLHEFile(unittest.TestCase):
```

Comment:
do we want to go with `unittest` or  just `pytest`  auto-discovery (I'm somewhat partial to the latter?)

Reply:
I would highly recommend this is all done in `pytest`.

Reply:
See [9d9998d](https://github.com/scikit-hep/pylhe/pull/29/commits/9d9998d2e40e754bf1333f719e1444de040b2249)